source/features/sticky-csv-header.css

```diff
@@ -0,0 +1,33 @@
+.markdown-body .csv-data {
```

Comment:
Does this apply only to CSV data and other tabular data?

Reply:
I think it does.